File: BD/Classe/Juge.php
<?php
/**
 *
 */
class Juge
{

  private $_NumJury;
  private $_NumGroupe;
  private $_IdHeure;
  private $_salle;

  function __construct($NumJury,$NumGroupe,$idHeure,$salle)
  {
    $this->_NumJury = $NumJury;
    $this->_NumGroupe = $NumGroupe;
    $this->_IdHeure = $idHeure;
    $this->_salle = $salle;
  }

  public function get_NumJury(){
    return $this->_NumJury;
  }

  public function get_NumGroupe(){
    return $this->_NumGroupe;
  }

  public function get_idHeure(){
    return $this->_IdHeure;
  }

  public function getSalle(){
    return $this->_salle;
  }

  public function setSalle($newSalle){
    $this->_salle = $newSalle;
  }

  public function setIdHeure($id)
  {
    $this->_IdHeure = $id;
  }

  public function __toString()
  {
      return $this->_NumGroupe. " ".$this->_IdHeure." ".$this->_NumJury."<br>";
  }
}


 ?>

File: MyApplication/app/src/main/java/com/example/olivet/myapplication/Juge.java
package com.example.olivet.myapplication;

/**
 * Created by fdubois on 17/01/18.
 */

public class Juge {

    private int NumJury;
    private int NumGroupe;
    private int idHeure;

    // Constructeur
    public Juge(int numJ, int numG, int idH) {
        this.NumJury=numJ;
        this.NumGroupe=numG;
        this.idHeure=idH;
    }


    public int getNumJury() {
        return NumJury;
    }

    public void setNumJury(int numJury) {
        NumJury = numJury;
    }

    public int getNumGroupe() {
        return NumGroupe;
    }

    public void setNumGroupe(int numGroupe) {
        NumGroupe = numGroupe;
    }

    public int getIdHeure() {
        return idHeure;
    }

    public void setIdHeure(int idHeure) {
        this.idHeure = idHeure;
    }
}
